application/wypl-core/src/main/java/com/wypl/wyplcore/calendar/CalendarServiceUtil.java

```diff
@@ -0,0 +1,192 @@
+package com.wypl.wyplcore.calendar;
+
+import java.time.DayOfWeek;
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.temporal.TemporalAdjusters;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.wypl.jpacalendardomain.calendar.data.RepetitionCycle;
+import com.wypl.jpacalendardomain.calendar.domain.Schedule;
+import com.wypl.wyplcore.schedule.data.response.ScheduleFindResponse;
+
+public class CalendarServiceUtil {
+
+	public static List<ScheduleFindResponse>  getScheduleResponses(Schedule schedule, LocalDate startDate, LocalDate endDate) {
+		if (!schedule.isRepetition()) return List.of(ScheduleFindResponse.of(schedule, schedule.getStartDateTime(), schedule.getEndDateTime()));
```

Comment:
```suggestion
		if (!schedule.isRepetition()) {
			return List.of(ScheduleFindResponse.of(schedule, schedule.getStartDateTime(), schedule.getEndDateTime()));
		}
```

이렇게 단 한줄이라도 if문이 있다면 다음과 같이 해주시면 좋을 것 같아요!

Reply:
그렇게 수정하겠습니다 감사합니다!